arch/inst/Zk/aes32dsmi.yaml

```diff
@@ -3,9 +3,9 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: aes32dsmi
-long_name: No synopsis available.
+long_name: AES middle round decryption instruction for RV32.
 description: |
-  No description available.
+  This instruction sources a single byte from `rs2` according to `bs`. To this it applies the inverse AES SBox operation, and a partial inverse MixColumn, before XOR'ing the result with `rs1`. This instruction must always be implemented such that its execution latency does not depend on the data being operated on.
```

Comment:
nit, optional: since I've asked for changes anyway, could you break this line up into shorter lines? The breaks in the spec are fine:
> This instruction sources a single byte from `rs2` according to `bs`. To this it applies the inverse AES
> SBox operation, and a partial inverse MixColumn, before XOR'ing the result with `rs1`. This
> instruction must always be implemented such that its execution latency does not depend on the
> data being operated on.